spec/uo_cli_spec.rb

```diff
@@ -1,11 +1,140 @@
 RSpec.describe URBANopt::CLI do
-  it 'has a version number' do
-    expect(URBANopt::CLI::VERSION).not_to be nil
+
+  test_directory = File.join("spec", "test_directory")
+  test_scenario = File.join(test_directory, "two_building_scenario.csv")
+  test_feature = File.join(test_directory, "example_project.json")
+  call_cli = "ruby #{File.join("lib", "uo_cli.rb")}"
+
+
+  # Ensure clean slate for testing
+  # +dir_or_file+ string - path to a file or folder
+  def delete_directory_or_file(dir_or_file)
+    if File.exists?(dir_or_file)
+      FileUtils.rm_rf(dir_or_file)
+    end
+  end
+  
+  
+  context "Admin" do
+    it 'has the correct version number' do
+      # This test forces devs to come into the test suite when they've made changes to the cli
+      expect(URBANopt::CLI::VERSION).to eq("0.2.3")
```

Comment:
Instead of this, I would check that the system call uo -v returns URBANopt::CLI::VERSION.  that way you don't have to mod the tests for each version and you are testing the CLI call.

Reply:
Done